privacyidea/static/components/login/controllers/loginControllers.js

```diff
@@ -423,7 +425,41 @@ angular.module("privacyideaApp")
 
     $scope.welcomeNext = function () {
         $scope.welcomeStep += 1;
-    }
+    };
+
+    $scope.reload = function() {
+        // emit a signal to the scope, that just listens
+        $scope.$broadcast("piReload");
+    };
+
+    $scope.checkReloadListeners = function () {
```

Comment:
Hmm, if I understand correctly this function is used to ensure that the reload button is only displayed if there are any reload listeners listening to the event. But does this happen?

Reply:
Yes. There are several sub-scopes like ``Tokendetail``. Currently there are not listeners defined.
The reload button will then reload the parent scope (the token list)...

Reply:
Ah, okay. But is it ever the case that the reload button is hidden because ``$scope.reloadListeners == 0``.

Reply:
Ah, nevermind, I somehow missed the ``TODO``.